cmd/podman/inspect.go

```diff
@@ -86,8 +90,40 @@ func inspectCmd(c *cli.Context) error {
 		inspectType = inspectTypeContainer
 	}
 
-	inspectedObjects, iterateErr := iterateInput(getContext(), c, args, runtime, inspectType)
+	if c.String("format") == formats.KubeString {
+		containers, err := getAllOrLatestContainers(c, runtime, -1, "")
```

Comment:
This whole section should probably be in it's own function.  This does not seem like it is well integrated into the inspect code.

I would prefer something like
outputFormat := c.String("format")
swtcih(outputFormat) {
case formats.KubeString:
                 output, err = kubeOutput(c, runtime)
case formats.JsonStirng
                 ....
}

Reply:
moved to its own subcommand